src/contracts/mainnet/AaveV2wETHIRPayload.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title wETH Interest Rate Curve - Ethereum v2
+ * @author Llama
+ * @notice Upgrade wETH interest rate on Ethereum v2 to match Ethereum v3 Liquidity Pool.
+ * Governance Forum Post: https://governance.aave.com/t/arfc-weth-wsteth-interest-rate-curve-ethereum-network/11372
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x9ae28e9c82c5fc0d24cf1df788094e959d99f906d11b89e455a60ee16b071d6f
+ */
+contract AaveV2wETHIRPayload is IProposalGenericExecutor {
```

Comment:
You probably want to match the name of this file to be `AaveV2ETHwETHIRPayload` so that it's consistent with the rest of the files in this folder.